src/main/java/dataLoader/csvDataLoader.java

```diff
@@ -0,0 +1,33 @@
+package dataLoader;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.StringReader;
+
+/**
+ * Simple class that parses data set from a csv file
+ * without knowing the headers of the file
+ * */
+
+public class csvDataLoader {
+
+    private String location = "/path/to/file.csv";
+    private Reader csvData = new StringReader(location);
+
+    public void parseCsvData( String csv) throws IOException {
+        
+        CSVParser parser = CSVParser.parse(csvData, CSVFormat.DEFAULT);
+
+        for (CSVRecord record: parser) {
```

Comment:
Hmmmm, this may become problematic. If we have ```N``` records and each record has ```M``` fields then this is ```O(N*M)``` so if ```M``` is ```O(N)``` this means quadratic complexity....ouch....

Reply:
I'm open to suggestions on this one, as well as, speed & resources issues. in any case I think we can worry about that later on.

Reply:
ok I agree, we should have a look at it later...more I wanted to underline this rather than anything else